egs/librispeech/asr1/RESULTS.md

```diff
@@ -1,3 +1,34 @@
+# pytorch large conformer-transducer with specaug + speed perturbation (4 GPUs)
+
+- Environments
+  - python version: `3.8.3 (default)  [GCC 7.3.0]`
+  - espnet version: `espnet 0.10.7a1`
+  - chainer version: `chainer 6.0.0`
+  - pytorch version: `pytorch 1.10.0`
+
+- Model files (archived to model.tar.gz by `$ pack_model.sh`)
+    - model link: ([pretrained model](https://drive.google.com/file/d/1LiJ8VYNHWyYocapBbjnQsKy5GuBLRSP-/view?usp=sharing))
+    - training config file: `conf/tuning/transducer/train_conformer-rnn_transducer.yaml`
+    - decoding config file: `conf/decode.yaml`
```

Comment:
Is it intended? Default decode config for LibriSpeech is linked to [decode_pytorch_transformer_large.yaml](https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/conf/tuning/decode_pytorch_transformer_large.yaml) which is used for CTC/Att model.

It's not an issue because only `beam-size` is taken into account and default arguments will be used, but the beam size should be decreased here (60 -> 10)

Reply:
Surprisingly, the result with beam size 10/60 are not different on this model.

Reply:
```suggestion
    - decoding config file: `conf/tuning/transducer/decode.yaml`
```